spec/win32/functional/win32/certstore_spec.rb

```diff
@@ -30,44 +40,50 @@
   describe "#get" do
     before { add_cert }
     let(:cert_pem) { File.read('.\spec\win32\assets\GlobalSignRootCA.pem') }
-
+    before(:each) do
+      allow_any_instance_of(certbase).to receive(:get_cert_pem).and_return(cert_pem)
+    end
     # passing valid thumbprint
     it "returns the certificate_object if found" do
       thumbprint = "b1bc968bd4f49d622aa89a81f2150152a41d829c"
-      expect(@store).to receive(:cert_get).with(thumbprint, store_location: CERT_SYSTEM_STORE_CURRENT_USER, store_name: "My").and_return(cert_pem)
-      @store.get(thumbprint, store_location: CERT_SYSTEM_STORE_CURRENT_USER, store_name: "My")
+      expect(@store).to receive(:cert_get).with(thumbprint).and_return(cert_pem)
+      @store.get(thumbprint)
     end
 
     # passing invalid thumbprint
-    it "returns nil if certificate not found" do
+    it "returns raises an Arugment error" do
```

Comment:
is this a potentially breaking change to the API?

Reply:
I undid those to be more permissive about thumbprint format.